mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_notification_config_test.go

```diff
@@ -0,0 +1,139 @@
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckSecurityCenterOrganizationNotificationConfigDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccSecurityCenterOrganizationNotificationConfig_basic(context),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_scc_v2_organization_notification_config.default", "description", "A test organization notification config"),
+					resource.TestCheckResourceAttr("google_scc_v2_organization_notification_config.default", "streaming_config.0.filter", "severity = \"HIGH\""),
+				),
+			},
+			{
+				ResourceName:      "google_scc_v2_organization_notification_config.default",
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
+					"parent",
+					"config_id",
+				},
+			},
+			{
+				Config: testAccSecurityCenterOrganizationNotificationConfig_update(context),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_scc_v2_organization_notification_config.default", "description", "An updated test organization notification config"),
+					resource.TestCheckResourceAttr("google_scc_v2_organization_notification_config.default", "streaming_config.0.filter", "severity = \"CRITICAL\""),
+				),
+			},
+		},
+	})
+}
+
+func testAccSecurityCenterOrganizationNotificationConfig_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_pubsub_topic" "scc_v2_organization_notification_config" {
+  name = "tf-test-topic-%{random_suffix}"
+}
+
+resource "google_scc_v2_organization_notification_config" "default" {
+  config_id    = "tf-test-config-%{random_suffix}"
+  organization = "%{org_id}"
+  location     = "global"
+  description  = "A test organization notification config"
+  pubsub_topic = google_pubsub_topic.scc_v2_organization_notification_config.id
+
+  streaming_config {
+    filter = "severity = \"HIGH\""
+  }
+}
+`, context)
+}
+
+func testAccSecurityCenterOrganizationNotificationConfig_update(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_pubsub_topic" "scc_v2_organization_notification_config" {
+  name = "tf-test-topic-%{random_suffix}"
+}
+
+resource "google_scc_v2_organization_notification_config" "default" {
+  config_id    = "tf-test-config-%{random_suffix}"
+  organization = "%{org_id}"
+  location     = "global"
+  description  = "An updated test organization notification config"
+  pubsub_topic = google_pubsub_topic.scc_v2_organization_notification_config.id
+
+  streaming_config {
+    filter = "severity = \"CRITICAL\""
+  }
+}
+`, context)
+}
+
+func testAccCheckSecurityCenterOrganizationNotificationConfigDestroyProducer(t *testing.T) resource.TestCheckFunc {
```

Comment:
This function can most likely be removed

Reply:
addressed